application/common/models/PasswordHistory.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace common\models;
+
+use common\helpers\Utils;
+use yii\helpers\ArrayHelper;
+
+class PasswordHistory extends PasswordHistoryBase
+{
+    public $password;
+
+    public function rules(): array
+    {
+        return ArrayHelper::merge([
+            [
+                'created_utc', 'default', 'value' => Utils::now(),
```

Comment:
yes, probably a worthwhile tweak.  The current implementation is relying on the fact that `created_utc` won't be provided but is required...it's easy enough to supplement that with a behavior.  The default will still be needed as well. https://github.com/silinternational/idp-id-broker/commit/8538d733d2160abb37b8ee405a38c988aa5fc118